### PR TITLE
Completed CSS Grid Classes | Cody Wiseman

### PR DESCRIPTION
**Pull request by codywiseman**
Screen Shot 2020-11-12 at 3 04 47 PM

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.container {
```
Comment (BrettAlbright) [inline css-grid-classes/styles.css]: the only thing missing here is the max-width property from the instructions. Please update!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
One really small update!

File: css-grid-classes/styles.css (hunk 1 above)
Comment (codywiseman) [reply css-grid-classes/styles.css]: Screen Shot 2020-11-12 at 8 15 34 PM

I made the updates to the max-width property. Let me know how everything looks!

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**